### PR TITLE
[LOOP-4414] Basal rate precision correction

### PR DESCRIPTION
**Pull request by nhamming**
use min to get the necessary number of decimal points (smaller will have more decimal points)

https://tidepool.atlassian.net/browse/LOOP-4414

### CODE REVIEW

File: LoopKitUI/Views/FractionalQuantityPicker.swift [hunk 1]
```diff
@@ -216,7 +216,7 @@ fileprivate extension Collection where Element == Decimal {
     /// - Precondition: The collection is sorted in ascending order.
     func deltaScale(boundedBy maxScale: Int) -> Int {
         let roundedToMaxScale = lazy.map { $0.rounded(toPlaces: maxScale) }
-        guard let maxDelta = roundedToMaxScale.adjacentPairs().map(-).map(abs).max() else {
+        guard let maxDelta = roundedToMaxScale.adjacentPairs().map(-).map(abs).min() else {
```
Comment (ps2) [inline LoopKitUI/Views/FractionalQuantityPicker.swift]: I was surprised that this was wrong, but I tested it, and I think you're right.  It should be using the smallest delta to determine the number of decimal places.

The name for the var should probably be changed to `minDelta`.  And there appears to be a copy of this in the watch app that should also be updated.

**Review by ps2 — CHANGES_REQUESTED** [review]

File: LoopKitUI/Views/FractionalQuantityPicker.swift (hunk 1 above)
Comment (nhamming) [reply LoopKitUI/Views/FractionalQuantityPicker.swift]: oh... right :)

Comment (nhamming) [reply LoopKitUI/Views/FractionalQuantityPicker.swift]: @ps2 just noticed your note about the watch app copy. Any hints as to where I can find this or what copy I'm looking for?

Comment (nhamming) [reply LoopKitUI/Views/FractionalQuantityPicker.swift]: oh I think I found it. See https://github.com/tidepool-org/Loop/pull/545

**Review by ps2 — APPROVED** [review]
LGTM!

**Pull request merged**